plugins/azure/eventGrid/domainPublicAccess.js

```diff
@@ -0,0 +1,51 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Domain Public Access',
+    category: 'Event Grid',
+    domain: 'Messaging services',
+    description: 'Ensure that Azure Event Grid domains do not have public access enabled.',
+    more_info: 'Enabling public access for Event Grid domains can expose sensitive information and increase the risk of unauthorized access.',
```

Comment:
```suggestion
    more_info: 'By default, domains are accessible from internet as long as the request comes with valid authentication and authorization exposing sensitive information. By disabling public access, Event Grid topics can be configured to use private endpoint.',
```

Reply:
the link provided mostly uses Event Grid topics instead of Event Grid domains, should we use domains or topics?